src/main/java/ti4/commands/uncategorized/InfoThreadCommand.java

```diff
@@ -0,0 +1,57 @@
+package ti4.commands.uncategorized;
+
+import java.util.List;
+
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import ti4.AsyncTI4DiscordBot;
+import ti4.helpers.Helper;
+import ti4.map.Game;
+import ti4.map.GameManager;
+import ti4.map.Player;
+import ti4.message.MessageHelper;
+
+public interface InfoThreadCommand {
```

Comment:
I like the interface - you can probably implement it on /ac info, /so info, /pn info, /relic info, etc.